includes/class-omise-rest-webhooks-controller.php

```diff
@@ -42,14 +42,14 @@ public function callback( $request ) {
 			return new WP_Error( 'omise_rest_wrong_header', __( 'Wrong header type.', 'omise' ), array( 'status' => 400 ) );
 		}
 
-		$body = json_decode( $request->get_body() );
+		$body = json_decode( $request->get_body(), true );
```

Comment:
Note: to make the payload data consistency on both Callback and Webhook data, this `json_decode` should be converting the payload data into array instead of object (as the same as OmiseCharge object).

This would help `Omise_Payment_Result` class to handle the same-format data.